mu-plugins/rest-api/endpoints/class-wporg-base-locale-banner-controller.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace WordPressdotorg\MU_Plugins\REST_API;
+
+/**
+ * Base_Locale_Banner_Controller
+ */
+class Base_Locale_Banner_Controller extends \WP_REST_Controller {
```

Comment:
```suggestion
abstract class Base_Locale_Banner_Controller extends \WP_REST_Controller {
```

To go with the later suggestion.